api/node-echoprint-server/controllers/api.js

```diff
@@ -100,3 +115,73 @@ exports.ingest = function(req, res) {
     });
   });
 };
+
+/**
+* Upload
+*
+* Validates and uploads files
+*
+* @param {Object} req
+* @param {Object} res
+*/
+function upload(req,res) {
```

Comment:
⚠️ I believe that I will never merge until this is sorted.

You have content. Willingness to remember.